site/styles/_footer.scss

```diff
@@ -49,6 +49,31 @@ footer {
   }
 }
 
+.footer-social-media-icons {
+  padding-top: 10px;
+  text-align: center;
+  @include media-breakpoint-up(xl) {
+    text-align: left;
+    :first-child {
```

Comment:
This isn't a good approach because it's applying these styles to _every_ first child under `footer-social-media-icons` which includes every `<i>`.

You also have some weird looking focus rings: 
![image](https://user-images.githubusercontent.com/10665859/220487785-986d2ebd-3387-40df-87cc-61f9bee215ad.png)

I think a better approach would be to use `grid` on the container with `gap` to easily handle margins between icons, and then have no margin or padding on the `<a>`s or `<i>`s.

Another thing:  That negative margin doesn't help because the icons have inconsistent widths in the font file.